04-spatial-operations.Rmd

```diff
@@ -676,10 +676,13 @@ if (knitr::is_latex_output()){
 
 Imagine that we need to join the `capacity` variable in `cycle_hire_osm` onto the official 'target' data contained in `cycle_hire`.
 This is when a non-overlapping join is needed.
-The simplest method is to use the topological operator `st_is_within_distance()`, as demonstrated below using a threshold distance of 20 m (note that this works with projected and unprojected data).
+The simplest method is to use the topological operator `st_is_within_distance()`, as demonstrated below using a threshold distance of 20 m.
```

Comment:
Another thought: is this more of a binary predicate than a topological operator? "Topological operator" implies to me that it's operating on, change, the topology of the geometry, whatever that means...

Reply:
Of course that comment applies equally to the old text as to your new text, just a rather late-in-the-day thought!

Reply:
> Another thought: is this more of a binary predicate than a topological operator? "Topological operator" implies to me that it's operating on, change, the topology of the geometry, whatever that means...

Yes, let's use binary, see https://r-spatial.org/book/03-Geometries.html#sec-de9im